### PR TITLE
Fix browser warning and footer containers.

### PR DESCRIPTION
**Pull request by davidlehn**
- Add container around browser warning.
- Move footer outside main content with container.  Footer has its own
  container and having it inside another one causes centering issues.

### CODE REVIEW

File: app-component.html [hunk 1]
```diff
@@ -9,10 +9,12 @@
   </div>
 
   <div ng-show="!$ctrl.route.changing" class="br-content">
-    <div ng-if="$ctrl.config.userAgent.obsolete" class="alert alert-danger">
-      Your browser ({{$ctrl.config.userAgent.family}}
-      {{$ctrl.config.userAgent.major}}) is <strong>out of date</strong>.
-      Please <a href="http://www.updateyourbrowser.net/">update your browser.</a>
+    <div ng-if="$ctrl.config.userAgent.obsolete" class="container">
```
Comment (mattcollier) [inline app-component.html:12]: I'm wondering if this `container` class should be `$root.app.ngClass.rootContainer || 'container'`, like on the new L20?

Comment (davidlehn) [reply app-component.html:12]: I'm not sure.  The footer is internally just using "container".  Is that an issue there too?

**Comment by davidlehn** [comment]
Also note that this moves the hr above footer (here or in the footer) to outside containers.  It'll span the whole window vs having edge margins.  I think that looks fine in this case.

**Comment by davidlehn** [comment]
This may need to be reworked a bit.  With the hr+footer outside the container, the footer's hr is wide and has more top margin (I think), but the hr before the demo warning will still be within the main container and have margins.  Looks odd in dev mode.

**Pull request closed without merging**